main.go

```diff
@@ -16,9 +17,13 @@ func main() {
 	defaultGpgPubringPath := "$GNUPGHOME/pubring.kbx or $HOME/.gnupg/pubring.kbx"
 
 	var verbose bool
+	var libnotify bool
+	var libnotifyDuration time.Duration
 	var u2fAuthPendingPath string
 	var gpgPubringPath string
 	flag.BoolVar(&verbose, "v", false, "print verbose output")
+	flag.BoolVar(&libnotify, "libnotify", false, "use libnotify for notifications")
+	flag.DurationVar(&libnotifyDuration, "libnotify-duration", 2*time.Second, "notify-send duration")
```

Comment:
Do we really need to provide this level of customization and override a default duration? Feels like too much detail for this tool, I'd remove this argument and just let user configure their desired durations in notification manager itself.

Reply:
Ok.